lib/gsup/protocol/gsup_msg.py

```diff
@@ -111,3 +111,13 @@ def get_ies_by_name(ie_name: str, message: dict):
             if ie_name in ie:
                 ies.append(ie)
         return ies
+
+    @staticmethod
+    def get_num_vectors_req(message: dict):
```

Comment:
Not sure that I would add this here especially as it hides logic of returning 5 per default if not specified or capping things to 5. That is IMHO not something that a util function should decide and I would personally move this into the AIR part of GSUP.